src/index.js

```diff
@@ -4,8 +4,9 @@ import fs from 'fs-extra'
 import path from 'path'
 
 export const topic = {
-  name: 'hello',
-  description: 'says hello (example plugin)'
+  name: 'ac',
```

Comment:
this will become `autocomplete`, but needs to not conflict with builtin stuff for now